### DISCUSSION TITLE
How to view office doc through libreoffice

### DISCUSSION THREAD

**Discussion by shubhkesh-happiestminds** [pimcore/pimcore#11176] [General]
Hello
I have istalled the libreoffice and imagic
(https://user-images.githubusercontent.com/92010212/149284746-41680cf8-5cb8-4b3f-ab6d-98f83ee03e70.png)
Can someone guide me how my office doc like ppt , xls  will open thourhg libreoffice. Currently doc is not being open and not getting any error

**Comment by fashxp** [comment]
Hi,
make sure symfony messenging queue is processed...

BR
Christian

**Reply by shubhkesh-happiestminds** [reply]
I am using pimcore 5.2 it is required symfony messenging queue ?
